vernac/vernac.mllib

```diff
@@ -9,6 +9,8 @@ Himsg
 Locality
 Egramml
 Vernacextend
+Declare
+DeclareHints
```

Comment:
For uniformity, this should probably be called `ComHints` instead.

Reply:
Done.